tests/integration/test_compress.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+from test_tools.generators import mk_seq_array
+
+import cunumeric as num
+from legate.core import LEGATE_MAX_DIM
+
+
+def test():
+    # test on the 1D array:
+    a = mk_seq_array(np, (10,))
+    a_num = num.array(a)
+    res = np.compress([2, 3, 6], a, axis=0)
+    res_num = num.compress([2, 3, 6], a_num, axis=0)
```

Comment:
Isn't the first argument supposed to be an array of bools?

Reply:
I like that this example tests the case where `len(condition) < len(array)`, I'm just wondering if it should be bools (or 0/1s instead).

Reply:
Yes, you are right, it should be bool. The test worked because cunumeric converted it to bool (all True), there was also a warning produced